File: PiwikTrackeriOSDemo/GoalsViewController.m
//
//  GoalsViewController.m
//  PiwikTracker
//
//  Created by Mattias Levin on 10/6/13.
//  Copyright (c) 2013 Mattias Levin. All rights reserved.
//

#import "GoalsViewController.h"
@import PiwikTrackerSwift;

@interface GoalsViewController ()
@end

@implementation GoalsViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  [[PiwikTracker sharedInstance] sendViews:@[@"menu", @"goals"]];
}

- (IBAction)sendGoalAction:(id)sender {
  // Track a conversion
    [[PiwikTracker sharedInstance] sendGoalWithId:1 revenue:5];  
}


@end

File: Example/ios/iOS Example/GoalsViewController.swift
//
//  GoalsViewController.swift
//  ios
//
//  Created by Cornelius Horstmann on 08.10.18.
//  Copyright © 2018 Mattias Levin. All rights reserved.
//

import UIKit
import MatomoTracker

class GoalsViewController: UIViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        MatomoTracker.shared.track(view: ["menu","goals"])
    }
    
    @IBAction func donateButtonTapped(_ sender: Any) {
       MatomoTracker.shared.trackGoal(id: 1, revenue: 5)
    }
    
}

